mos-platform/nes-action53/prg-rom-64.ld

```diff
@@ -0,0 +1,15 @@
+INCLUDE _prg-rom-banked.ld
+
+__prg_rom_size = 64;
+
+OUTPUT_FORMAT {
```

Comment:
As far as I can tell, it isn't necessary to break out separate linker scripts for 32 and 64 PRG-ROM sizes; the symbol value setting should be enough. If `__prg_rom_size == 32`, the `prg_rom_1` and `prg_rom_2` sections will have length zero, so the `FULL()` statements below won't add any content for them. That would let the default linker script PROVIDE() a default value for `__prg_rom_size`, and the user could override it just by defining a value for that symbol, either in inline assembly or a `settings.s` file.